src/applications/ivc-champva/10-10D/containers/IntroductionPage.jsx

```diff
@@ -98,41 +92,14 @@ const IntroductionPage = props => {
         </va-process-list-item>
       </va-process-list>
 
-      {!isLoggedIn ? (
-        <VaAlert status="info" visible uswds>
-          <h2>Sign in now to save time and save your work in progress</h2>
-          <p>Here’s how signing in now helps you:</p>
-          <ul>
-            <li>
-              We can fill in some of your information for you to save you time.
-            </li>
-            <li>
-              You can save your work in progress. You’ll have 60 days from when
-              you start or make updates to your application to come back and
-              finish it.
-            </li>
-          </ul>
-          <p>
-            <strong>Note:</strong> You can sign in after you start your
-            application. But you’ll lose any information you already filled in.
-          </p>
-          <p className="vads-u-margin-top--2">
-            <Link onClick={handleClick} to={pageList[1]?.path}>
-              Start your form without signing in
-            </Link>
-          </p>
-        </VaAlert>
-      ) : (
-        <SaveInProgressIntro
-          formId={formConfig.formId}
-          headingLevel={2}
-          prefillEnabled={formConfig.prefillEnabled}
-          messages={formConfig.savedFormMessages}
-          pageList={pageList}
-          startText="Start the Application"
-        />
-      )}
-
+      <SaveInProgressIntro
+        formId={formConfig.formId}
+        headingLevel={2}
+        prefillEnabled={formConfig.prefillEnabled}
+        messages={formConfig.savedFormMessages}
+        pageList={pageList}
+        startText="Start the Application"
+      />
```

Comment:
Removed this unnecessary, hand-rolled SIP alert and just used the default one. Did this in the other two forms as well.